<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QMainWindow" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ATPRase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
    <item>
     <layout class="QVBoxLayout" name="ItemLableList">
      <item>
       <widget class="ItemLabel" name="label_1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ItemLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ItemLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ItemLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ItemLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ItemLabel" name="label_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="log">
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
              <pointsize>48</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>ATPRase</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <widget class="QPushButton" name="button_loadADP">
               <property name="geometry">
                <rect>
                 <x>200</x>
                 <y>60</y>
                 <width>140</width>
                 <height>40</height>
                </rect>
               </property>
               <property name="text">
                <string>加载ADP</string>
               </property>
              </widget>
              <widget class="QPushButton" name="button_settingATPRun">
               <property name="geometry">
                <rect>
                 <x>200</x>
                 <y>120</y>
                 <width>140</width>
                 <height>40</height>
                </rect>
               </property>
               <property name="text">
                <string>配置ATP运行环境</string>
               </property>
              </widget>
              <widget class="QPushButton" name="button_help">
               <property name="geometry">
                <rect>
                 <x>200</x>
                 <y>180</y>
                 <width>140</width>
                 <height>40</height>
                </rect>
               </property>
               <property name="text">
                <string>帮助</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemLabel</class>
   <extends>QLabel</extends>
   <header location="global">ItemLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
